napari_arboretum/graph.py

```diff
@@ -117,7 +117,7 @@ def build_subgraph(layer, node):
     -------
     root_id : int, None
         The root node ID of the tree which contains the node.
-    nodes : list
+    nodes : list[TreeNode]
```

Comment:
can you add the appropriate typing above? I think here it would be `tuple[Union[Optional[int]], list[TreeNode]]`

Reply:
you've added it in docstring but not the signature yeah (in my view it only needs to be one, but I think the signature nicer for `mypy`)

Reply:
👍 done